components/application-broker/internal/broker/broker.go

```diff
@@ -106,9 +110,13 @@ func New(applicationFinder appFinder,
 	serviceInstanceGetter serviceInstanceGetter,
 	emLister listers.ApplicationMappingLister,
 	brokerService *NsBrokerService,
+	appClient *appCli.Interface,
 	mClient *mappingCli.Interface,
+	kClient kubernetes.Interface,
```

Comment:
Where do you use the application operator client and Kubernetes client?

Reply:
It was an unused import. Fixed :)